.github/workflows/tests.yml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, 3.10.0-rc.1]
+        python-version: [3.7, 3.8, 3.9, 3.10.0-rc.1, "3.11-dev"]
```

```suggestion
        python-version: [3.7, 3.8, 3.9, "3.10", "3.11-dev"]
```
https://dev.to/hugovk/the-python-3-1-problem-85g